prisma/migrations/20221006163340_create_discussions_table3/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- DropForeignKey
+ALTER TABLE "Discussion" DROP CONSTRAINT "Discussion_authorId_fkey";
+
+-- AddForeignKey
+ALTER TABLE "Discussion" ADD CONSTRAINT "Discussion_authorId_fkey" FOREIGN KEY ("authorId") REFERENCES "users"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
`SetNull` sounds like a better option because once the user delete their account, we can display the user details as `Deleted account` but show the comment content at the same time.